lib/model/query/forms.js

```diff
@@ -149,6 +145,21 @@ select id from form;`,
         .orderBy(db.raw('coalesce(forms."name", forms."xmlFormId")'), 'asc')
         .then(map(unjoin))),
 
+  // TODO: temporarily copy+pasta from _get below pending better resolution
+  // of this transaction nightmare. (also, vestigial options)
```

Comment:
Does "vestigial options" means that no callers set options at this time? Asking because the old implementation respected options passed in and this one does not.

Reply:
more that no options are used. but it's sort of the standard call arg structure rn. but also everything to do w queries needs to be really rethought. i'm not sure what better way to do it exists in js.